packages/react/src/components/ListBuilder/ListBuilder.mdx

```diff
@@ -38,19 +38,54 @@ import { ListBuilder } from 'carbon-addons-iot-react';
+| selectedDefaultExpandedIds | arrayOf                     |                | An array of the ids of the selected items that should be expanded                                                                                      |
+| selectedEditingStyle       | 'single' or 'single-nesting | undefined      | The editing style of the hirerchy list showing the selected items                                                                                      |
+| selectedItems              | arrayOf                     | []             | An array of all selected items                                                                                                                         |
+| showLoaderInAvailableList  | boolean                     | false          | If true the list for available columns will show a loader only                                                                                         |
+| showLoaderInSelectedList   | boolean                     | false          | If true the list for selected columns will show a loader only                                                                                          |
+| testId                     | string                      | 'list-builder' |                                                                                                                                                        |
+| useCheckboxes              | boolean                     | false          | If true checkboxes will be used for selection so selected items can stay in the left list                                                              |
+| i18n                       | object                      |                | (see [i18n prop](#i18n-prop))                                                                                                                          |
+
+### i18n Prop
+
+| Name                              | Type     | Default                                  | Description                                            |
+| :-------------------------------- | :------- | :--------------------------------------- | :----------------------------------------------------- |
+| addLabel                          | string   | 'Add item to list'                       | Label for the add button                               |
+| allListEmptyText                  | string   |                                          | Message shown if the all items list is empty           |
+| allListSearchPlaceholderText      | string   | 'Enter a value to search all items'      | Placeholder text for the search box for all items      |
+| allListTitle                      | function | (count) => `Items (${count} available)`  | Title for the list of all items                        |
+| clearSearchIconDescription        | string   | xxx                                      | Icon description label for the clear search icon       |
```

Comment:
The xxx was an old copy paste leftover, it was suppose to be empty. I left plenty of defaults untouched (empty) since they also have defaults in the subcomponents in which they are used.